docker-compose.yml

```diff
@@ -4,6 +4,8 @@ postgres:
     - "PG_USER=panoptes"
     - "DB=panoptes_development"
     - "PASS=panoptes"
+  ports:
```

Comment:
do you need this on by default, i run two db services both on port 5432 but one in docker. The docker one won't start if the port is already taken